samples/mnedc/client-config.yaml

```diff
@@ -0,0 +1,2 @@
+server-ip: 192.168.0.125
```

Comment:
As @suresh-lc's [comment](https://github.com/lf-edge/edge-home-orchestration-go/issues/209#issuecomment-762714187), put this file in the `configs`.
Also, can you add some helpful comments on the first line like [LINK]?(https://github.com/envoyproxy/envoy/blob/main/configs/terminate_http1_connect.yaml)

Reply:
I will create the configs and examples in the next PR and move it.

> Also, can you add some helpful comments on the first line like [LINK]?

Yes, that is a good idea. I will do that.